int32[] short_vector
